app/models/broadcast_announcement.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: broadcast_announcements
+#
+#  id              :bigint           not null, primary key
+#  expiry          :date
+#  link            :text
+#  message         :text
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint
+#  user_id         :bigint           not null
+#
+class BroadcastAnnouncement < ApplicationRecord
+  belongs_to :user
+  belongs_to :organization, optional: true
+  validates :link, format: URI::DEFAULT_PARSER.make_regexp(%w[http https]), allow_blank: true
+  validates :message, presence: true
+
+  def expired?
+    return false if expiry.nil?
+    expiry < Time.zone.today
+  end
+
+  def self.filter_announcements(parent_org)
+    BroadcastAnnouncement.where(organization_id: parent_org)
```

Comment:
To include "no-org" (admin) announcements, this should be `where(organization_id: [parent_org, nil].uniq)`.

Reply:
Hmm.   [This comment is largely an aside to @dorner]  I know this work is going to be the basis for the next step -- which is the admin announcements -- but/and the initial model only has   admin announcements going down to the banks.    I'm not saying don't make this change, but that we may have to think about a broadcast level when we implement the next step.

Reply:
Ok, you do want the future admin announcements to be displayed on the partner dashboard? I was thinking the future admin announcements would only be for the banks

Reply:
There's a case for admins being able to communicate to the partners, but it would be only some announcements .   I think it's out of scope for this issue. @dorner?  Are you ok with that?

Reply:
Ah OK - yeah we can leave it as is then and address the admin stuff later.